### ISSUE TITLE
Labels moved

### ISSUE THREAD

**Issue by dpenas** [jiahuang/d3-timeline#68]
With the latest released version (0.0.5) the graph's labels are not shown. After downloading the newest changes those labels do appear, but they are moved a few pixels to the bottom, whereas the previous version (0.0.4) did work correctly. 
Is there something I need to consider using the newest version or should still work the same?

**Comment by inodb** [comment]
Yeah looks like the latest release has some issues. Things seem to be alright in this commit:

http://rawgit.com/jiahuang/d3-timeline/081e83b4926708b9e1822aa3d3e5e5541a5abc84/examples/example.html

**Comment by marcbowes** [comment]
2f1e5bc8f4ea93e396b1622a07c6e83f35fb62bc seems to have introduced it.
